api/src/pcapi/routes/backoffice_v3/collective_offers.py

```diff
@@ -95,11 +94,8 @@ def _get_collective_offers(
         if search_query.isnumeric():
             base_query = base_query.filter(educational_models.CollectiveOffer.id == int(search_query))
         else:
-            name_query = search_query.replace(" ", "%").replace("-", "%")
-            name_query = clean_accents(name_query)
-            base_query = base_query.filter(
-                sa.func.unaccent(educational_models.CollectiveOffer.name).ilike(f"%{name_query}%")
-            )
+            name_query = "%{}%".format(search_query)
+            base_query = base_query.filter(educational_models.CollectiveOffer.name.ilike(f"%{name_query}%"))
```

Comment:
```suggestion
            base_query = base_query.filter(educational_models.CollectiveOffer.name.ilike(name_query))
```
tu mets deux fois les `%`